src/main/java/fi/helsinki/cs/tmc/core/communication/ExerciseDownloader.java

```diff
@@ -126,10 +159,12 @@ public boolean handleSingleExercise(Exercise exercise, String path) {
         downloadExerciseZip(exercise.getZipUrl(), filePath.toString());
         try {
             taskExecutor.extractProject(filePath, Paths.get(path));
-        } catch (IOException e) {
+        }
+        catch (IOException e) {
             log.error("Could not extract archive: {}", path.toString());
             return false;
-        } finally {
+        }
```

Comment:
Keep indentation as it was; no new lines after blocks. More at http://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s4.1.2-blocks-k-r-style